src/components/NavBar/NavBar.js

```diff
@@ -29,8 +29,12 @@ const NavBar = () => {
         <NavLink onClick={handleClick} className="nav-link" exact to="/">
           Home
         </NavLink>
-        <NavLink onClick={handleClick} className="nav-link" to="/incidents">
-          Incidents
+        <NavLink
+          onClick={handleClick}
+          className="nav-link"
+          to="/incident-reports"
```

Comment:
I do like the new pathing, makes it more clear but however i do we might need to add documentation